packages/stylex/src/StyleXSheet.js

```diff
@@ -280,54 +289,123 @@ export class StyleXSheet {
   /**
    * Insert a rule into the stylesheet.
    */
-  insert(rawLTRRule: string, priority: number, rawRTLRule: ?string) {
-    // Inject the stylesheet if it hasn't already been
-    if (this.injected === false) {
+  insert(
+    rawLTRRule: string,
+    priority: number,
+    rawRTLRule: ?string,
+    metadata?: ?{|
+      constKey?: string,
+      constVal?: string | number,
+    |},
+  ): void {
+    // 1) ensure the stylesheet exists
+    if (!this.injected) {
       this.inject();
     }
 
-    if (rawRTLRule != null) {
+    // 2) defineConsts update: patch or register constant values
+    if (
+      rawLTRRule === '' &&
+      metadata?.constKey != null &&
+      metadata.constVal != null
+    ) {
+      const key: string = (metadata.constKey: any);
+      const newVal: string = String((metadata.constVal: any));
```

Comment:
I thought we had moved to `as` syntax? This should be:

```suggestion
      const key: string = metadata.constKey as any;
      const newVal: string = String(metadata.constVal as any);
```

However, if you're not seeing a syntax error with your existing code we might have issue with the `.flowconfig`.